Neos.Media/Classes/Domain/Repository/AssetRepository.php

```diff
@@ -284,7 +284,9 @@ protected function addAssetVariantFilterClause(Query $query): void
 
         $variantClassNames = $this->reflectionService->getAllImplementationClassNamesForInterface(AssetVariantInterface::class);
         foreach ($variantClassNames as $variantClassName) {
-            $queryBuilder->andWhere('e NOT INSTANCE OF ' . $variantClassName);
+            if (is_a($variantClassName, AssetInterface::class)) {
```

Comment:
i think we should use 

`in_array($interface, class_implements($value), true)`

@nezaniel pointed out to me the flaws of `is_a` but i forgot ^^